library/src/main/java/org/apache/apex/malhar/lib/state/spillable/SpillableSetMultimapImpl.java

```diff
@@ -63,13 +107,14 @@
   @NotNull
   private SpillableMapImpl<Slice, Pair<Integer, V>> map;
```

Comment:
Have you explored the possibility of changing the key type from Slice to something else so you don't have to store extra eight bytes for the Slice?